pkg/e2e/verify/pod_webhook.go

```diff
@@ -45,6 +45,7 @@ var _ = ginkgo.Describe(podWebhookTestName, func() {
 					"system:authenticated",
 				},
 			})
+			time.Sleep(5 * time.Second)
```

Comment:
Is there a more deterministic way to prevent this failure than a sleep? :thinking:

Reply:
@aliceh If your hypothesis RE: selector sync sets is true, could we instead directly query whether that sync operation is complete? Does the sync set have a status field that could tell us?